apps/src/templates/sectionAssessments/StudentMultipleChoiceOverviewTable.jsx

```diff
@@ -0,0 +1,159 @@
+import React, {Component, PropTypes} from 'react';
+import {Table, sort} from 'reactabular';
+import {tableLayoutStyles, sortableOptions} from "../tables/tableConstants";
+import i18n from '@cdo/locale';
+import wrappedSortable from '../tables/wrapped_sortable';
+import orderBy from 'lodash/orderBy';
+import FontAwesome from '@cdo/apps/templates/FontAwesome';
+
+export const COLUMNS = {
+  NAME: 0,
+  NUM_MULTIPLE_CHOICE_CORRECT: 1,
+  NUM_MULTIPLE_CHOICE: 2,
+  PERCENT_CORRECT: 3,
+  SUBMISSION_TIMESTAMP: 4,
+};
+
+const studentOverviewDataPropType = PropTypes.shape({
+  id: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
+  numMultipleChoiceCorrect: PropTypes.number.isRequired,
+  numMultipleChoice: PropTypes.number.isRequired,
+  percentCorrect: PropTypes.string.isRequired,
+  submissionTimestamp: PropTypes.string,
```

Comment:
it looks like this is the only place you're using `submissionTimestamp` instead of `submissionTimeStamp`, so i don't think this proptype is doing anything 😢 

i've usually seen it as `timestamp` rather than `timeStamp`, so i would change it to that, but that's obviously a bigger change and non-blocking